libetrv/utils.py

```diff
@@ -1,4 +1,4 @@
-from collections import Iterable
+from collections.abc import Iterable
```

Comment:
Will this approach work with older versions of Python? Library was designed to run on Raspberry Pi OS, which was using pretty old version of python